src/Pos/Wallet/Web/Server/Methods.hs

```diff
@@ -332,7 +332,7 @@ updateUserProfile profile = setProfile profile >> getUserProfile
 
 getWallet :: WalletWebMode ssc m => CAddress -> m CWallet
 getWallet cAddr = do
-    balance <- getBalance =<< decodeCAddressOrFail cAddr
+    balance <- fmap mkCCoin . getBalance =<< decodeCAddressOrFail cAddr
     meta <- getWalletMeta cAddr >>= maybe noWallet pure
     pure $ CWallet cAddr balance meta
   where
```

Comment:
As far as I see, frontend's `send` and `sendExtended` were modified. Why aren't they here?